src/portable/st/stm32_fsdev/dcd_stm32_fsdev_pvt_st.h

```diff
@@ -83,42 +83,85 @@
 
 // For purposes of accessing the packet
 #if ((PMA_LENGTH) == 512u)
-#  define PMA_STRIDE (2u)
+const size_t PMA_STRIDE = 2u;
```

Comment:
please either use macros or enum, I still prefer those two for pure constant expression. We could worry about the compiler complain later.